draft-ietf-tls-esni.md

```diff
@@ -194,6 +196,12 @@ Upon receiving the server's response, the client determines whether ECH was
 accepted or rejected and proceeds with the handshake accordingly. (See
 {{client-behavior}} for details.)
 
+Informally, a primary goal of ECH is ensuring that connections to servers in the
+same anonymity set are indistinguishable from one another for passive and active
+attackers, modulo traffic analysis, without any existing security properties of
```

Comment:
"modulo traffic analysis": I would suggest cutting this and leaving it to Security Considerations. It's a matter that a sophisticated service provider might try to address, so we don't want to exclude this here.

Reply:
That works for me!